pyrealm/core/water.py

```diff
@@ -148,7 +148,7 @@ def calc_density_h2o(
             functions are numerically unstable.
 
     Returns:
-        Water density as a float in (g cm^-3)
+        Water density in kg m3.
```

Comment:
Not sure I understand? This is a wrapper around one of the _fisher or _chen methods, both of which return kg m3, so this does too.

Reply:
OK. I just thought as it says it returns a density it should be km/m3, as for example in line 120

Reply:
Oh - wait. You mean I should be using `kg/m3` or  something like `kg m^-3`, rather than what I have? If so, yup - sloppy on my part.

The typo above for `km/m3` had me completely baffled 🤣

Reply:
Sorry, yes, kg/m3 I meant! Sorry for the typo